### ISSUE TITLE
API 2.0 Release Candidate Guinea Pigs and Feedback

### ISSUE THREAD

**Issue by petems** [pearkes/tugboat#182]
Now #180 is merged, I want to have people do some rough testing to make sure I've not broken any existing workflows too much with the switch to the 2.0 API.

Any volunteers?

Potential Candidates: @alekseyp, @oryband, @zlanich, @FredrikWendt, @edanisko, @FreedomBen, @conorsch

Instructions:

```
gem uninstall tugboat
gem install tugboat -v 2.0.0.RC1
tugboat authorize
```

Pending no huge hiccups, the final version of the gem will be released 26th October and all the issues related to API 2.0 will be closed.

**Comment by alekseyp** [comment]
Awesome news!

1) get install should be for -v 2.0.0.RC1
2) if you trying to ssh to a droplet that was part of returned search by name, it won't use your ssh key.
If you use full name for droplet it will.


```
➜  ~ tugboat ssh server
Droplet fuzzy name provided. Finding droplet ID...Multiple droplets found.

0) a.server ()
1) s01.server ()
2) s02.server ()
3) a01.server ()
4) u01.server ()

Please choose a droplet: ["0", "1", "2", "3", "4"] 0
Executing SSH a.server...
Password:
fish: Job 1, 'tugboat ssh el33t.net' terminated by signal SIGINT (Quit request from job control (^C))
➜  ~ tugboat ssh a.server
Droplet fuzzy name provided. Finding droplet ID...done, xxxx (a.server)
Executing SSH (a.server)...
Welcome to Ubuntu 14.04.3 LTS (GNU/Linux 3.13.0-24-generic x86_64)
...
root@a:~# logout
Connection to 188.226.206.164 closed.
➜  ~ 

```

**Comment by FreedomBen** [comment]
I've had no issues with halting, resizing, starting, destroying, and creating.  I did have an issue getting the create to use my SSH key, but it seems like user error atm.

**Comment by tjcrandall** [comment]
I'm seeing an issue specifying a region. It's still expecting numeric Region IDs rather than the region slug.
```tyler@lenny:~$ tb version
Tugboat 2.0.0.RC1
tyler@lenny:~$ tb create tor-test --region tor1
Expected numeric value for '--region'; got "tor1"
tyler@lenny:~$ tb create tor-test --region 11
Queueing creation of droplet 'tor-test'...Failed to create Droplet: You specified an invalid region for Droplet creation.```

**Comment by petems** [comment]
@tjcrandall Good catch, I'll fix that now.

**Comment by petems** [comment]
@tjcrandall Fixed in https://github.com/pearkes/tugboat/pull/186

**Comment by tjcrandall** [comment]
@petems :+1:

**Comment by petems** [comment]
@FreedomBen The SSH issue might be related to https://github.com/pearkes/tugboat/issues/192, I'll see if I can get a fix out for that.

**Comment by petems** [comment]
Closed by #183
